<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1286</width>
    <height>757</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Cutelooker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/images/Cutelooker.png</normaloff>:/images/Cutelooker.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1286</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionLoad_JSON"/>
    <addaction name="actionLoad_Log_JSON"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Log"/>
    <addaction name="actionInformation"/>
   </widget>
   <widget class="QMenu" name="menu_Options">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="actionPlot_pagefile"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuView"/>
   <addaction name="menu_Options"/>
  </widget>
  <action name="actionLoad_JSON">
   <property name="text">
    <string>Load &amp;Data</string>
   </property>
  </action>
  <action name="actionLoad_Log_JSON">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load &amp;Log</string>
   </property>
  </action>
  <action name="action_Log">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Log</string>
   </property>
  </action>
  <action name="actionInformation">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Information</string>
   </property>
  </action>
  <action name="actionPlot_pagefile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Plot &amp;pagefile</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
